Gruntfile.js

```diff
@@ -412,9 +414,8 @@ module.exports = function(grunt) {
 
   grunt.registerTask("komanda-package", function(platforms) {
     try {
-    fs.mkdirSync("package/");
-    } catch(e) {
-    }
+      fs.mkdirSync("package/");
+    } catch(e) {}
```

Comment:
I wouldn't catch the error here, if the package folder creation fails, the rest of the code is likely to fail too and since the error is swallowed, debugging that simple problem will be harder.